client/src/modules/accounts/edit/accounts.edit.modal.html

```diff
@@ -50,7 +50,7 @@
     <!-- @TODO Re-disable account type editing after accounts are stable -->
     <!-- Account type read only -->
     <!-- <div ng-if="!AccountEditCtrl.isCreateState" class="form-group"> -->
-      <!-- <label class="control-label">{{ "FORM.LABELS.ACCOUNT_TYPE" | translate }}</label> -->
+      <!-- <label class="control-label">{{ "FORM.LABELS.ACCOUNT_TYPE"  }}</label> -->
```

Comment:
Eventually we should re-enable this: #2149.  Glad to see the code is still here!